bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -515,7 +515,7 @@ public Widget getWidget(Sitemap sitemap, String id) {
                     for (int i = 2; i < id.length(); i += 2) {
                         w = ((LinkableWidget) w).getChildren().get(Integer.valueOf(id.substring(i, i + 2)));
                     }
-                } catch (NumberFormatException e) {
+                } catch (NumberFormatException | IndexOutOfBoundsException e) {
```

Comment:
Alright, I will change it. Though the `NumberFormatException` was caught here I thought it doesn't hurt to catch the other one too...